### PR TITLE
Spelling grammar

### PR DESCRIPTION
**Pull request by bdfinst**
## Description

MD Linting
Correct sidebar menu titles for Zarf commands
Improve tone
Correct spelling issues
Correct grammar issues

### CODE REVIEW

**Comment by YrrepNoj** [comment]
@bdfinst When you get the chance, can you run the make target `make docs-and-schema`. That command will auto-generate all of the docs based off the Go struct schemas.

(If pre-commit gives you any formatting complaints please ignore them, the pipeline is testing for any diffs against the same make target.)

**Comment by bdfinst** [comment]
> @bdfinst When you get the chance, can you run the make target `make docs-and-schema`. That command will auto-generate all of the docs based off the Go struct schemas.
> 
> (If pre-commit gives you any formatting complaints please ignore them, the pipeline is testing for any diffs against the same make target.)
@YrrepNoj - Yep, done. I had to write a script to modify the files after generation to correct the sidebar menus.

File: docs/10-roadmap.md [hunk 1]
```diff
@@ -4,18 +4,15 @@
...
+If you want to add bug reports or feature requests to be considered on our roadmap, please feel free to [add an issue](https://github.com/defenseunicorns/zarf/issues) on the GitHub repository.
 
-<br />
 
 ## GA Release
-Right now, Zarf is still in its 'beta' phase. We are working on a few final things before we release the official 1.0 general availability (GA) release. The current planned date for the GA release is mid June 2022. The work still needed for the GA release can be found on the roadmap with [this filter](https://github.com/orgs/defenseunicorns/projects/1/views/1?filterQuery=repo%3A%22defenseunicorns%2Fzarf%22+milestone%3A%22zarf+ga%22).
-
 
+Right now, Zarf is still in its 'beta' phase. We are working on a few final things before we release the official 1.0 general availability (GA) release. The work still needed for the GA release can be found on the roadmap with [this filter](https://github.com/orgs/defenseunicorns/projects/1/views/1?filterQuery=repo%3A%22defenseunicorns%2Fzarf%22+milestone%3A%22zarf+ga%22).
```
Comment (Madeline-UX) [inline docs/10-roadmap.md:14]: This link does not take them to the information that the sentence implies @jeff-mccoy is there another place we want the user to go for updates to v1.0 or should we just list out the goals the team wants to meet before cutting v1.0 - currently documented in coda

File: docs/12-support.md [hunk 2]
```diff
@@ -1,6 +1,9 @@
 # Support
 
-1. Make sure you've read the [Zarf Overview](./0-zarf-overview.md), [Understanding the Basics](./1-understand-the-basics.md), and the [Getting Started](./3-getting-started.md) guides. 
-2. Look for an answer in [the frequently asked questions](./9-faq.md).
+1. Make sure you've read the guides in:
```
Comment (Madeline-UX) [inline docs/12-support.md:3]: This is much better thank you.

File: docs/3-getting-started.md [hunk 3]
```diff
@@ -4,7 +4,7 @@ Welcome to the documentation about Zarf, the air-gap tool! Let's get you started
 
 ## Installing Zarf
 
-There are multiple ways to get the Zarf CLI onto your machine including installing from the Defense Unicorns Homebrew Tap, downloading a prebuilt binary from our GitHub releases, or even building the CLI from scratch yourself.
+There are multiple ways to get the Zarf CLI onto your machine including installing the Defense Unicorns Homebrew Tap, downloading a prebuilt binary from our GitHub releases, or even building the CLI from scratch yourself.
```
Comment (Madeline-UX) [inline docs/3-getting-started.md:7]: Can you turn this into a list as well  (with links if applicable)
- defense Unicorns Homebrew tap
- downloading prebuilt binary from Github release
- building the CLI from source

File: docs/3-getting-started.md [hunk 4]
```diff
@@ -42,26 +44,25 @@ You can learn more about building [here](./4-user-guide/1-the-zarf-cli/1-buildin
 
 ## Verifying Zarf Install
 
-Now that you have installed Zarf onto your path, let's verify that it is working by checking two things! First, we'll check the version of Zarf that has been installed with the command:
```
Comment (Madeline-UX) [inline docs/3-getting-started.md:45]: This is a lot clearer!

File: docs/3-getting-started.md (hunk 3 above)
Comment (bdfinst) [reply docs/3-getting-started.md]: Done with included links

File: docs/10-roadmap.md (hunk 1 above)
Comment (Madeline-UX) [reply docs/10-roadmap.md:14]: A decision was made with the team to move the roadmap to READ.me in a different issue. Will create a separate issue for updating the read me to include the road map and then make updating the linked part of that PR

**Review by Madeline-UX — APPROVED** [review]

**Review by jeff-mccoy — APPROVED** [review]

**Pull request merged**